### PR TITLE
New package: termdown-1.16.0

### LINKED ISSUE #14182: Package request: termdown
https://github.com/trehn/termdown

> Countdown timer and stopwatch in your terminal

Very useful CLI-based timer/stopwatch/alarm with feedback. As far as I know, nothing similar currently exists in Void repos.

### PR DESCRIPTION
**Pull request by singul0**
Also a python-pyfiglet, termdown's dependency, that is absent in repository

closes #14182

### CODE REVIEW

**Comment by bra1nwave** [comment]
Please put `python-figlet` in a separate commit

File: srcpkgs/termdown/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'termdown'
+pkgname=termdown
+version=1.16.0
+revision=1
+archs=noarch
+build_style=python3-module
+pycompile_module="termdown"
+hostmakedepends="python3-setuptools"
+depends="python3 python3-dateutil python3-click python3-pyfiglet"
```
Comment (bra1nwave) [inline srcpkgs/termdown/template]: You can remove `python3` since it's pulled in by the other python dependencies

Comment (singul0) [reply srcpkgs/termdown/template]: Did it

File: srcpkgs/python-pyfiglet/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	vlicense LICENSE
+	vdoc README
+}
+
+python3-pyfiglet_package() {
+	archs=noarch
+	pycompile_module="pyfiglet"
+	depends="python3"
+	pkg_install() {
+		vmove usr/lib/python3*
```
Comment (bra1nwave) [inline srcpkgs/python-pyfiglet/template]: Please fix indentation and put `usr/lib/python3*` in double quotes

Comment (singul0) [reply srcpkgs/python-pyfiglet/template]: Done

**Comment by Chocimier** [comment]
I wouldn't add python2 version of pyfiglet. If you want it, move executable to right package, make alternative and set different short_desc, see other python templates.

termdown's license is -only and we do not include GPL into package.

File: srcpkgs/termdown/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'termdown'
+pkgname=termdown
+version=1.16.0
+revision=1
+archs=noarch
+build_style=python3-module
+pycompile_module="termdown"
+hostmakedepends="python3-setuptools"
+depends="python3-dateutil python3-click python3-pyfiglet"
```
[the remark this answers is not in the document]
Comment (bra1nwave) [reply srcpkgs/termdown/template]: [No, it doesn't](https://github.com/trehn/termdown/blob/a70ac7c17b0f9e3260cdcec00725903d20e0ef7f/setup.py#L34)

[the remark this answers is not in the document]
Comment (Chocimier) [reply srcpkgs/termdown/template]: Can you actually run it without python3-setuptools installed? I can't.

[the remark this answers is not in the document]
Comment (bra1nwave) [reply srcpkgs/termdown/template]: You're right. I should've checked first. Sorry

File: srcpkgs/python3-pyfiglet/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+pkgname=python3-pyfiglet
+version=0.8.0
+revision=1
+archs=noarch
+wrksrc="pyfiglet-${version}"
+build_style="python3-module"
+pycompile_module="pyfiglet"
+hostmakedepends="python3-setuptools"
+depends="python3"
+short_desc="Implementation of figlet written in Python "
```
Comment (Chocimier) [inline srcpkgs/python3-pyfiglet/template]: trailing space

File: srcpkgs/termdown/template [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'termdown'
+pkgname=termdown
+version=1.16.0
+revision=1
+archs=noarch
+build_style=python3-module
+pycompile_module="termdown"
```
Comment (Chocimier) [inline srcpkgs/termdown/template]: termdown.py

File: srcpkgs/termdown/template [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools python3-dateutil python3-click python3-pyfiglet"
+checkdepends=$depends
```
Comment (Chocimier) [inline srcpkgs/termdown/template]: Nice, caring of tests is quite rare. But please brace and quote.

Comment (singul0) [reply srcpkgs/termdown/template]: I've actually added test in the recent changes. Is it OK to use screen to test apps, that requires terminal capabilities?

Comment (Chocimier) [reply srcpkgs/termdown/template]: Well, `screen false` returns success, so it is not useful. Default do_check for that package has 0 tests, but it imports module, what is non-negligible test in as dynamic language as python is.

**Comment by singul0** [comment]
I give up on tests for now, as I didn't managed to find a sane way to run termdown without terminal capabilities.

**Pull request merged**